### DISCUSSION TITLE
[bug] REST API for releases missing Link

### DISCUSSION THREAD

**Discussion by henryiii** [community/community#24328] [Apps, API and Webhooks]
<p>The rest API is missing a Link in the header; even though it had one up until about a day ago. This has broken the setup-cmake action, which used to use the “next” in the Link to access pages. Try something like:</p>
<pre><code class="lang-bash">curl -I "https://api.github.com/repos/Kitware/CMake/releases"
</code></pre>
<p>There is no Link in the header.</p>
<p>See <a href="https://github.com/jwlawson/actions-setup-cmake/pull/22" class="inline-onebox" rel="noopener nofollow ugc">chore: bump checkout version by henryiii · Pull Request #22 · jwlawson/actions-setup-cmake · GitHub</a> for the broken TypeScript-based action, which in turn has broken the CI for pybind11 and other packages.</p>

**Comment by cocreature** [comment]
<p>We are encountering the same issue on <a href="https://github.com/digital-asset/daml" class="inline-onebox" rel="noopener nofollow ugc">GitHub - digital-asset/daml: The DAML smart contract language</a>.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome! <a class="mention" href="/cocreature">@cocreature</a> and :wave: <a class="mention" href="/henryiii">@henryiii</a></p>
<p>Thanks so much for letting us know. It should be fixed now - can you let us know if you’re still seeing problems with this?</p>

**Comment by cocreature** [comment]
<p>Thanks <a class="mention" href="/canuckjacq">@canuckjacq</a>! The issue has been fixed for us now and we get the expected response including pagination headers again.</p>

**Comment by henryiii** [comment]
<p>Fantastic, thank you! Yes, it started working again yesterday.</p>
